src/summary/cards/vertical-label-value.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./vertical-label-value.css";
+
+export default function VerticalLabelValue(props: VerticalLabelValueProps) {
+  return (
+    <div className={styles.root}>
+      <label className={`omrs-type-body-small ${styles.label}`}>
+        {props.label}
+      </label>
+      <div style={props.valueStyles}>{props.value || "\u2014"}</div>
```

Comment:
Maybe give it a `// em-dash if no value` or even just  `// em-dash` ?

Reply:
done